### PR TITLE
feat: show total choice voting power in proposal results

### PR DESCRIPTION
**Pull request by ChaituVR**
### Summary

Closes: #515

- Add voting power beside each choice
 
  Untitled 4

### How to test

1. Go to different proposals, and you should see total voting power on each choice of result
2. http://localhost:8080/#/s:safe.eth/proposal/0x8d97075eabfdf3fb9f43f3b490e8e5a7326d77bee919b78543621e36e5b2d233
3. http://localhost:8080/#/sn:0x07c251045154318a2376a3bb65be47d3c90df1740d8e35c9b9d943aa3f240e50/proposal/1/
4. http://localhost:8080/#/sn:0x07c251045154318a2376a3bb65be47d3c90df1740d8e35c9b9d943aa3f240e50/proposal/1/

### CODE REVIEW

**Comment by bonustrack** [comment]
We shouldn't show the symbol, take too much space.

**Comment by ChaituVR** [comment]
> We shouldn't show the symbol, take too much space. Otherwise it looks good

I thought you missed it 😆 removed it now 
Untitled 4

**Review by bonustrack — APPROVED** [review]
tACK, let's wait for @Sekhmet approval to merge

File: apps/ui/src/components/ProposalResults.vue
Comment (Sekhmet) [inline apps/ui/src/components/ProposalResults.vue:75]: We already have `decimals` prop that we can use.

**Review by Sekhmet — APPROVED** [review]
tACK

CI failing unrelated to this PR.

**Pull request merged**
